File: hsg9-tinh/2023-2024-hatinh-ts10/sums/sums.cpp
#include <iostream>

using namespace std;

#define inputFile "sums2.inp"
#define outputFile "sums2.out"

typedef long long int lli;

lli n;
lli S;


void input()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    
    freopen(inputFile, "r", stdin);
    cin >> n;
}


void process()
{
    S = n * (n + 2);
}


void output()
{
    freopen(outputFile, "w", stdout);

    cout << S;
}


int main()
{
    input();
    process();
    output();

    return 0;
}

File: hsg9-tinh/2023-2024-hatinh-ts10/sums/sums.py
import os
import sys

input_file = os.path.join(sys.path[0], 'sums.inp')
output_file = os.path.join(sys.path[0], 'sums.out')

n = 0


def input():
    global n
    
    with open(input_file) as f:
        n = int(f.readline().strip())


def process():
    global n, S
    
    S = n * (n + 2)


def output():
    global S

    with open(output_file, 'w') as f:
        f.write(str(S))


if __name__ == '__main__':
    input()
    process()
    output()